lib/robin-hood-hashing missing from source tarballs

Probably because it is a Git submodule.

The source looks good, but now I have a `cmake` surprise

```
dh_auto_configure -- -DSEQAN3_NO_LEMON=OFF
	cd obj-x86_64-linux-gnu && DEB_PYTHON_INSTALL_LAYOUT=deb cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON -DFETCHCONTENT_FULLY_DISCONNECTED=ON -DCMAKE_INSTALL_RUNSTATEDIR=/run -DCMAKE_SKIP_INSTALL_ALL_DEPENDENCY=ON "-GUnix Makefiles" -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu -DSEQAN3_NO_LEMON=OFF ..

-- You can run `make` to build the application.
-- Found Doxygen: /usr/bin/doxygen
-- You can run `make doc` to build api documentation.
-- You can run `make test` to build and run tests.
CMake Error at /usr/share/cmake-3.27/Modules/CPack.cmake:685 (message):
  CPack license resource file: "/dev/shm/needle/LICENSE.md" could not be
  found.
Call Stack (most recent call first):
  /usr/share/cmake-3.27/Modules/CPack.cmake:690 (cpack_check_file_exists)
  CMakeLists.txt:61 (include)
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,7 +3,7 @@ cmake_minimum_required (VERSION 3.10)
 ## CUSTOMISE
 
 # Define the application name and version.
-project (needle VERSION 1.0.0)
+project (needle VERSION 1.0.3)
 
 ## BUILD
 
@@ -47,3 +47,42 @@ add_subdirectory (doc EXCLUDE_FROM_ALL)
 
 enable_testing ()
 add_subdirectory (test EXCLUDE_FROM_ALL)
+
+## PACKAGE
+# Change version in this file: project (needle VERSION x.x.x)
+# Change version in src/main.cpp: parser.info.version = "x.x.x";
+# To package, create a clean directory:
+#
+# mkdir needle-package
+# cd needle-package
+# git clone https://github.com/seqan/needle
+# cd needle
+# # git checkout VERSION # optional: Checkout a tag if it exists.
+# git submodule update --init
+# cd lib/seqan3
+# git submodule update --init
+# cd ../../../
+# mkdir package
+# cd package
+# cmake ../needle
+# cmake --build . --target package_source
+#
+# Will create needle-[VERSION]-Source.tar.xz{,.sha256}.
+# We do git submodule update --init instead of a recursive clone, because we do not want to package the recursive
+# submodules, but only what we need. The script handles all submodules at the time of writing, but newer submodules
+# might be missing.
+# This should be done in a clean directory, because everything in the needle checkout will be packaged, i.e., also
+# local build directories if they exist.
+
+set (CPACK_GENERATOR "TXZ")
+
+set (CPACK_PACKAGE_VERSION "${needle_VERSION}")
+set (CPACK_PACKAGE_VENDOR "seqan")
+set (CPACK_PACKAGE_CHECKSUM "SHA256")
+set (CPACK_RESOURCE_FILE_LICENSE "${needle_SOURCE_DIR}/LICENSE.md")
+set (CPACK_RESOURCE_FILE_README "${needle_SOURCE_DIR}/README.md")
+
+set (CPACK_SOURCE_GENERATOR "TXZ")
+set (CPACK_SOURCE_IGNORE_FILES "\\\\.git($|/)")
+
+include (CPack)

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -156,7 +156,7 @@ int run_needle_estimate(seqan3::argument_parser & parser)
     estimate_ibf_arguments args{};
     estimate_arguments estimate_args{};
     parser.info.short_description = "Estimate expression value of transcript based on the Needle index.";
-    parser.info.version = "1.0.0";
+    parser.info.version = "1.0.3";
     parser.info.author = "Mitra Darvish";
 
     args.path_out = "expressions.out";
@@ -505,7 +505,7 @@ int main(int argc, char const ** argv)
     needle_parser.info.description.push_back("Needle allows you to build an Interleaved Bloom Filter (IBF) with the "
                                              "command ibf or estimate the expression of transcripts with the command "
                                              "estimate.");
-    needle_parser.info.version = "1.0.0";
+    needle_parser.info.version = "1.0.3";
     needle_parser.info.author = "Mitra Darvish";
 
     try

